src/lib/seam/components/AccessCodeDetails/AccessCodeDetails.tsx

```diff
@@ -12,20 +12,25 @@ import { useAccessCode } from 'lib/seam/access-codes/use-access-code.js'
 import { AccessCodeDevice } from 'lib/seam/components/AccessCodeDetails/AccessCodeDevice.js'
 import { DeviceDetails } from 'lib/seam/components/DeviceDetails/DeviceDetails.js'
 import { Alerts } from 'lib/ui/Alert/Alerts.js'
+import { Button } from 'lib/ui/Button.js'
 import { ContentHeader } from 'lib/ui/layout/ContentHeader.js'
 import { useIsDateInPast } from 'lib/ui/use-is-date-in-past.js'
 
 export interface AccessCodeDetailsProps {
   accessCodeId: string
   disableLockUnlock?: boolean
+  disableEdit?: boolean
```

Comment:
I know this looks redundant, but talking with @dawnho we realized there will be a common set of "global" config like this, so we want to keep the prop name the same for all the components that use them.

```suggestion
  disableAccessCodeEdit?: boolean
```

Reply:
However, we can't add this prop in this PR, it must be hard coded false as in the other component since we do not want to anything to use the access code form until it's generally released.